### DISCUSSION TITLE
[Question]: When I use curl to get the response, it get 404 not found.

### DISCUSSION THREAD

**Discussion by thugbobby** [BerriAI/litellm#1692] [General]
### What happened?

A bug happened!
When I use the curl like this: 
```
curl --location --request POST 'http://0.0.0.0:8000/chat/completions' \
--header 'Content-Type: application/json' \
--data-raw '{
                    "model": "qwen-72b-chat",
                    "stream": false,
                    "messages": [
                        {
                        "role": "user",
                        "content": "编写一段python代码，实现冒泡法排序，请完善注释，用代码块输出代码"
                        }
                    ]
                    }'
```
if the `stream` is `true`, then it work normal, but when the `stream` is false, it will response 404 like this:
![image](https://github.com/BerriAI/litellm/assets/68416779/e91b7dfe-f491-4c3b-ae69-20c90b19fe65)

**Comment by ghost** [comment]
404 means the endpoint wasn't found. Are you sure it's running on the right port?

**Reply by thugbobby** [reply]
I`m sure， when  I open the stream， it can work normal.
